src/guiguts/spell.py

```diff
@@ -0,0 +1,325 @@
+"""Search/Replace functionality"""
+
+import logging
+import os
+import regex as re
+
+from guiguts.checkers import CheckerDialog
+from guiguts.maintext import maintext, FindMatch
+from guiguts.preferences import preferences
+from guiguts.utilities import IndexRowCol, IndexRange
+
+logger = logging.getLogger(__package__)
+
+DEFAULT_DICTIONARY_DIR = os.path.join(
```

Comment:
This may not work if GG is installed as a package. See https://stackoverflow.com/a/20885799 for a way that should work for both methods, specifically https://docs.python.org/3.11/library/importlib.resources.html

We should see if there is anything we need to do with the poetry config such that these files are included in the final package too.

Reply:
By the way, do you think I should be testing out `poetry publish` at this stage or can I put that off for now if `build` appears to be working?